Formula/opencoarrays.rb

```diff
@@ -0,0 +1,54 @@
+class Opencoarrays < Formula
+  desc "open-source coarray Fortran ABI, API, and compiler wrapper"
+  homepage "http://opencoarrays.org"
+  url "https://github.com/sourceryinstitute/opencoarrays/releases/download/1.8.2/OpenCoarrays-1.8.2.tar.gz"
+  sha256 "dc5b98a62de85e38500f431b05290415d4a4a87001c0ba03fbe57403ad62010b"
+
+  head "https://github.com/sourceryinstitute/opencoarrays.git"
+
+  option "without-test", "Skip build time tests (not recommended)"
+
+  depends_on "gcc"
+  depends_on :fortran
+  depends_on :mpi => :cc
+  depends_on "cmake" => :build
+
+  fails_with :gcc
+  fails_with :gcc_4_0
+  fails_with :gcc_4_2
```

Comment:
Do I still need these? We do want a relatively modern C compiler... but I don't know that this is still necessary... won't `depends_on "gcc"` mean that GCC is linked and we're using clang or brewed gcc?